src/emulator/gxm/src/gxm.cpp

```diff
@@ -672,6 +674,7 @@ GLenum translate_internal_format(SceGxmTextureFormat src) {
             return GL_INTENSITY8;
         default:
         {
+            LOG_WARN("Unsupported translate internal format: {:#08X}", src);
```

Comment:
This shouldn't be here, I removed it for a reason: gxmtextlinear alread warns and with this it's smapped in the log...